device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceResourceSpec.scala

```diff
@@ -30,18 +34,22 @@ class DeviceResourceSpec extends ResourcePropSpec {
   import GeneratorOps._
 
   private val deviceNumber = DeviceRepository.defaultLimit + 10
+  private implicit val exec = system.dispatcher
+  private val publisher = DeviceSeenListener.action(MessageBusPublisher.ignore)
 
   def isRecent(time: Option[Instant]): Boolean = time match {
     case Some(t) => t.isAfter(Instant.now.minus(3, ChronoUnit.MINUTES))
     case None => false
   }
 
+  private def sendDeviceSeen(uuid: Uuid, lastSeen: Instant = Instant.now()): Unit =
+      publisher(DeviceSeen(defaultNs, uuid, Instant.now()))
```

Comment:
Ah yeah, that's better! Means you don't get annoying bugs when you forget the `eventually`